src/sections/Navbar.html

```diff
@@ -1,27 +1,27 @@
 <nav
-  class="flex flex-row justify-between items-center md:px-8 p-4 lg:p-6 xl:p-6 font-outfit lg:mb-12 sticky top-0 z-10"
+   class="top-0 z-10 sticky flex flex-row justify-between items-center lg:mb-12 md:px-8 p-4 lg:p-6 xl:p-6 font-outfit"
 >
-  <div class="flex flex-col md:text-2xl lg:text-2xl">
-    <p>DE</p>
-    <p>EN</p>
-  </div>
-  <div class="lg:absolute lg:pl-[40%]">
-    <h1 class="font-semibold text-2xl md:text-5xl">Baumeister</h1>
-  </div>
-  <div
-    class="absolute font-bold text-3xl space-y-10 md:space-y-24 lg:space-y-0 lg:flex flex-col lg:items-end hidden md:ml-[20%] lg:ml-0 pt-[120%] md:pt-[70%] md:text-5xl md:font-bold lg:pt-14 pl-32 lg:pl-[87%] lg:font-normal lg:text-2xl text-green-700 lg:text-black"
-    id="menu"
-  >
-    <p>About</p>
-    <p>Services</p>
-    <p>References</p>
-    <p>Contact</p>
-  </div>
+   <div class="flex flex-col md:text-2xl lg:text-2xl">
```

Comment:
This md:text-2xl will also work on larger screens, you can remove this lg:text-2xl